arch/src/aarch64/mod.rs

```diff
@@ -79,12 +79,10 @@ pub fn configure_vcpu(
 
```

Comment:
Could you please elaborate in the commit message the reason why we can get the hidden RAM back.

Reply:
Done.